### PR TITLE
feat: use description in all details views

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- adjust all static views to new metadata

**Related issue(s)**

https://github.com/kyma-project/busola/issues/2753

### CODE REVIEW

File: src/components/HelmReleases/HelmReleaseDescription.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import React from 'react';
+
+export const description = (
+  <Trans i18nKey={'helm-releases.description'}>
+    <ExternalLink
+      className="bsl-link"
+      url="https://helm.sh/docs/glossary/#release"
+    />
+  </Trans>
+);
```
Comment (mrCherry97) [inline src/components/HelmReleases/HelmReleaseDescription.js:12]: All of this should be moved to one generic component which you will import and pass just `url` and `i18nKey` prop.

File: src/resources/ClusterRoleBindings/ClusterRoleBindingDescription.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Trans } from 'react-i18next';
+import { Link as ReactSharedLink } from '../../shared/components/Link/Link';
```
Comment (mrCherry97) [inline src/resources/ClusterRoleBindings/ClusterRoleBindingDescription.js:2]: Adjust this to all of the rest of changed files.
```suggestion
import { Link as ReactSharedLink } from 'shared/components/Link/Link';
```

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
Annotations should be in short version (adjusted in [ResourceList PR](https://github.com/kyma-project/busola/pull/2755/files#diff-50e7b79247558ef2966389199851083ea7d321a8a8f88718138f8d3ff2249244R37) )
example: api-gateway cron-job
image

File: src/components/HelmReleases/HelmReleaseDescription.js (hunk 1 above)
Comment (dbadura) [reply src/components/HelmReleases/HelmReleaseDescription.js]: done

**Review by akucharska — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**